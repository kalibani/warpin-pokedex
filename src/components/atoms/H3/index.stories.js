// Stories H3 Component
// --------------------------------------------------------
import React, { Fragment } from 'react';
import { storiesOf } from '@storybook/react';
import H3 from './index';
import Notes from './readme.md';

storiesOf('Atoms', module).add('H3', () => (
  <H3 />
), {
  info: { inline: true, header: false, text: Notes }
});
