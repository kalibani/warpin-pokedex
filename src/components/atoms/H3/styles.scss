/* ==========================================================================
Styles for H3 component
========================================================================== */

// import mixin and variables
@import "../../../assets/styles/variables";

// Start styles H3 component
.a-h3 {
  font-size: 28px;
  font-weight: 600;
  color: $white;
  margin: 0;
}
