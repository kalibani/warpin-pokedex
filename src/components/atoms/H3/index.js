// H3 Component
// --------------------------------------------------------

import React from 'react';
import PropTypes from 'prop-types';
import './styles.scss';

const H3 = ({ propsName }) => (
  <tag>
    Functional Component (Stateless Component)
  </tag>
);

H3.propTypes = {
  propsName: PropTypes.string
};

H3.defaultProps = {
  propsName: ''
};

export default H3;
