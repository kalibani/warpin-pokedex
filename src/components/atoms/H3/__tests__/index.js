// Test H3 Component
// --------------------------------------------------------

import React from 'react';
import H3 from '../../index';
import renderer from 'react-test-renderer';

test('H3 is match the snapshot', () => {
const component = renderer.create(
H3,
);
let tree = component.toJSON();
expect(tree).toMatchSnapshot();
});
