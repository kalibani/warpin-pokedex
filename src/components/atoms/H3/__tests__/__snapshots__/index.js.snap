// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H3 is match the snapshot 1`] = `null`;
